apps/allocations/app/components/App/PayoutsTables.js

```diff
@@ -1,35 +1,32 @@
 import PropTypes from 'prop-types'
 import React from 'react'
 import styled from 'styled-components'
-import { Text, theme, Viewport, Badge } from '@aragon/ui'
-import { FieldTitle } from '../Form'
 
+import { Badge, Text, theme } from '@aragon/ui'
 
+const PayoutsTable = ({ list: List, data, ...listProps }) =>
+  <div>
+    <Text.Block size="large" weight="bold" style={{ marginBottom: '10px' }}>
+      Allocations
+      {' '}
+      <Badge.Info>{data.length}</Badge.Info>
+    </Text.Block>
 
-const PayoutsTable = props => {
-  const List = props.list
-
-  return (
-    <div>
-      <Text.Block size="large" weight="bold" style={{ marginBottom: '10px' }}>
-        Allocations
-        {' '}
-        <Badge.Info>{props.data.length}</Badge.Info>
-      </Text.Block>
-
-      <List {...props} />
-    </div>
-  )
-}
+    <List {...listProps} />
+  </div>
 
 PayoutsTable.propTypes = {
   data: PropTypes.arrayOf(PropTypes.object).isRequired,
+  list: PropTypes.node.isRequired,
+  listProps: PropTypes.object.isRequired,
 }
 
 const PayoutDescription = styled(Text.Block)`
   display: block;
+  /* stylelint-disable-next-line */
   display: -webkit-box;
   -webkit-line-clamp: 2;
+  /* stylelint-disable-next-line */
   -webkit-box-orient: vertical;
```

Comment:
We don't need these lines; styled-components adds backward-compatible styles for us. No need to disable the lines; just remove them.